docs/docusaurus/docs/core/installation_and_setup/components_how_to_configure_a_validation_result_store_in_amazon_s3/_update_your_configuration_file_to_include_a_new_store_for_validation_results_on_s.mdx

```diff
@@ -0,0 +1,58 @@
+```
+
+As shown in the previous example, you need to change the default ``store_backend`` settings to make the Store work with S3.  The ``class_name`` is set to ``TupleS3StoreBackend``, ``bucket`` is the address of your S3 bucket, and ``prefix`` is the folder in your S3 bucket where Validation Results are located.
+
+The following example shows the additional options that are available to customize `TupleS3StoreBackend`:
+
+```yaml title="File contents: great_expectations.yml"
+class_name: ValidationsStore
+store_backend:
+  class_name: TupleS3StoreBackend
+  bucket: '<your_s3_bucket_name>'
+  prefix: '<your_s3_bucket_folder_name>'  # Bucket and prefix in combination must be unique across all stores
+  boto3_options:
+    endpoint_url: ${S3_ENDPOINT} # Uses the S3_ENDPOINT environment variable to determine which endpoint to use.
+    region_name: '<your_aws_region_name>'
+```
+In the previous example, the Store name is ``validations_S3_store``. If you use a personalized Store name, you must also update the value of the `validations_store_name` key to match the Store name. For example:
+
+```python title="Python" name="version-0.17.23 docs/docusaurus/versioned_docs/version-0.17.23/snippets/aws_cloud_storage_pandas.py set_new_validations_store"
+```
+When you update the `validations_store_name` key value, Great Expectations uses the new Store for Validation Results.
+
+Add the following code to `great_expectations.yml` to configure the IAM user:
+
+```yaml title="File contents: great_expectations.yml"
+class_name: ValidationsStore
+store_backend:
+  class_name: TupleS3StoreBackend
+  bucket: '<your_s3_bucket_name>'
+  prefix: '<your_s3_bucket_folder_name>' # Bucket and prefix in combination must be unique across all stores
+  boto3_options:
+    aws_access_key_id: ${AWS_ACCESS_KEY_ID} # Uses the AWS_ACCESS_KEY_ID environment variable to get aws_access_key_id.
+    aws_secret_access_key: ${AWS_ACCESS_KEY_ID}
+    aws_session_token: ${AWS_ACCESS_KEY_ID}
+```
+
+Add the following code to `great_expectations.yml` to configure the IAM Assume Role:
+
+```yaml title="File contents: great_expectations.yml"
+class_name: ValidationsStore
+store_backend:
+  class_name: TupleS3StoreBackend
+  bucket: '<your_s3_bucket_name>'
+  prefix: '<your_s3_bucket_folder_name>' # Bucket and prefix in combination must be unique across all stores
+  boto3_options:
+    assume_role_arn: '<your_role_to_assume>'
+    region_name: '<your_aws_region_name>'
+    assume_role_duration: session_duration_in_seconds
+```
+
+:::caution
+If you are also storing <TechnicalTag tag="expectation" text="Expectations" /> in S3 [How to configure an Expectation store to use Amazon S3](../configure_expectation_stores.md), or DataDocs in S3 [How to host and share Data Docs](../../configuring_data_docs/host_and_share_data_docs.md), then make sure the ``prefix`` values are disjoint and one is not a substring of the other.
```

Comment:
These links don't exist in the current docs, hence the broken links.  It might be worth it to go through and just remove all of the existing links, then do a read through and re-insert links where it makes sense using the /core/ docs instead.

Reply:
In fact, I think I strongly recommend that: go through these component files and remove all of the links.  Then read through the doc and insert new links using the /core/ docs or otherwise adjust the text when it references guides that don't exist anymore.  That should resolve a lot of the broken link errors.

Reply:
@Rachel-Reverie  - done